src/main/cpp/teleopcontrol/TeleopControlFunctions.h

```diff
@@ -36,20 +36,16 @@ class TeleopControlFunctions
         AUTO_TURN_BACKWARD,
         AUTO_ALIGN_LEFT,
         AUTO_ALIGN_RIGHT,
-        AUTO_ALIGN_CENTER,
         AUTO_ALIGN_HUMAN_PLAYER_STATION,
         AUTO_ALIGN_BARGE,
         AUTO_ALIGN_PROCESSOR,
         AUTO_ALIGN_ALGAE,
```

Comment:
Are the deleted functions not used anymore; some of these I thought were implemented in the current code base